pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -18,16 +21,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ python makeWrapper zip pandoc ];
 
-  patchPhase = "rm youtube-dl";
-
-  configurePhase = ''
-    makeFlagsArray=( PREFIX=$out SYSCONFDIR=$out/etc PYTHON=${python}/bin/python )
```

Comment:
Losing the definition of `SYSCONFDIR=$out/etc` here has no adverse affect on the generated binary?